### ISSUE TITLE
Complete a search from the UI

### ISSUE THREAD

**Issue by blackpan2** [moltimate/moltimate-frontend#11]
Blocked by: moltimate/moltimate-backend#13
Size Estimation: M

From the UI complete a UI request and display the results. 

Acceptance Criteria:
* UI receives user input and makes a request of the backend
* UI receives the results and displays them. (This does not have to be the finalized format of the results if it does not, make a follow-up task)

**Comment by smcintosh881** [comment]
@blackpan2 this is done in that a search can be submitted, but there I need to update and include the motif upload. Im going to make a separate story for that because its a change in requirements and not small enough to be a subtask of this.
